whatwg.org/faq

```diff
@@ -50,8 +50,20 @@ governance structure for the WHATWG, together forming a
 
 <h3 id="spell-and-pronounce">How do you spell and pronounce WHATWG?<a class="self-link" href="#spell-and-pronounce"></a></h3>
 
-<p>It is spelled WHATWG, all uppercase, no spaces. It has various pronunciations: what-wee-gee,
-what-wig, what-double-you-gee.
+<p>It is spelled WHATWG, all uppercase, no spaces. It has various pronunciations:
+<dl>
+ <dt>what-wee-gee
+ <dd>en-US: /wɑːt.wi.dʒi/
+ <dd>en-UK: /wɒt.wi.dʒi/
```

Comment:
This goes for all three, but is there actually a difference in vowel length between en-US/en-GB here? Because I don't believe there is?